mtags/src/main/scala-3/scala/meta/internal/pc/InferredTypeProvider.scala

```diff
@@ -79,8 +79,8 @@ final class InferredTypeProvider(
         printer.typeString(short)
     /*
      * Get the exact position in ValDef pattern for val (a, b) = (1, 2)
-     * Suprisingly, val ((a, c), b) = ((1, 3), 2) will be covered by Bind
-     * https://github.com/lampepfl/dotty/issues/12627
+     * val ((a, c), b) = ((1, 3), 2) will be covered by Bind
+     * This is needed to Scala versions pre 3.1.2.
```

Comment:
:thinking: I'm wondering how this function call is avoided for versions greater than 3.1.2.

Reply:
I don't think this will get invoked for versions 3.1.2 and later since this should not go to that branch. Besides, this isn't really an issue, since the code is only invoked by the user in a code action.